plugin/web/extensions/infobox/core/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+<title></title>
+<style>
+  html, body{
+    background: transparent;
+    width: 100%;
+    height: 100%;
+    min-height: 500px;
```

Comment:
Is this going to break in mobile?

Reply:
I think it's fine since infobox has a built-in scroll wrapper outside.